### PR TITLE
Upgrade test-worker to use new Python for testing

### LINKED ISSUE #813: pylint pre-submit python 3.5.2
/kind bug

The pylint pre-submit is using python 3.5.2 but our python libraries are >= python 3.6

### PR DESCRIPTION
**Pull request by jinchihe**
**What this PR does / why we need it**:

I updated the test-worker image that's used Python 3.8, and removed some code that upgrading Python from 3.5 to 3.6 in test script. Thanks.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #813

### CODE REVIEW

**Comment by jinchihe** [comment]
Need clear up test env and retest later.

```
fatal: could not create leading directories of '/mnt/test-data-volume/kubeflow-kfserving-presubmit-e2e-819-7aa6c55-2176-41be/src/kubeflow
/kfserving': No space left on device
```

File: test/scripts/pylint-test.sh [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+update-alternatives --install /usr/bin/python3 python3 /usr/bin/python3.5 1
+update-alternatives --install /usr/bin/python3 python3 /usr/local/bin/python3.6 2
+# Work around the issue https://github.com/pypa/pip/issues/4924
+mv /usr/bin/lsb_release /usr/bin/lsb_release.bak
+
+echo "Installing requirement pacakges ..."
+python3 -m pip install --upgrade pip
+pip3 install --upgrade pylint
+
+python -m kubeflow.testing.test_py_lint --artifacts_dir=$1 --src_dir=$2
```
Comment (jlewi) [inline test/scripts/pylint-test.sh:43]: See comment 
https://github.com/kubeflow/kfserving/issues/813#issuecomment-627324759

We already have a test worker image with python 3.8. So you could probably just use that.

Comment (jinchihe) [reply test/scripts/pylint-test.sh:43]: Thanks @jlewi , Will try use new image.

**Comment by jinchihe** [comment]
/assign @cliveseldon 

I updated the test-worker image that's used Python 3.8, and removed some code that upgrading Python from 3.5 to 3.6 in test script. Thanks.

File: test/scripts/pylint-test.sh (hunk 1 above)
Comment (cliveseldon) [reply test/scripts/pylint-test.sh]: Did we not want to try this?

**Pull request merged**